VERSIONING.md

```diff
@@ -91,8 +91,9 @@ The following versions have been evaluated by third party security consultants p
 | [v3.3.0](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v3.3.0)          | 15 July 2021      | Penetration test evaluating (1) external attack surface, (2) ability to exfiltrate data from the system, (3) ability to transfer data between SREs, (4) ability to escalate privileges on the SRD. | No major security issues identified. |
 | [v3.4.0](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v3.4.0)          | 22 April 2022     | Penetration test evaluating (1) external attack surface, (2) ability to exfiltrate data from the system, (3) ability to transfer data between SREs, (4) ability to escalate privileges on the SRD. | No major security issues identified. |
 | [v4.0.0](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v4.0.0)          | 2 September 2022  | Penetration test evaluating ability to infiltrate/exfiltrate data from the system.                                                                                                                 | No major security issues identified. |
-| [v5.0.0-rc1](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1) | 18 September 2023 | Penetration test evaluating ability to infiltrate/exfiltrate data from the system. Testing next codebase, using Python and Pulumi.                                                                        | No major security issues identified. |
+| [v5.0.0-rc1](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1) | 18 September 2023 | Penetration test evaluating ability to infiltrate/exfiltrate data from the system. Testing next codebase, using Python and Pulumi.                                                                 | No major security issues identified. |
```

Comment:
Can we add 5.0.0-rc2 here too? Or can do in another PR if preferred.

Reply:
Right, this and the other one are actually changes Jim made in the `release-v5.0.0` branch, which this was based on, and I assume they're showing up because i put develop as the target branch instead of `release-v5.0.0`